cmd/operator-controller/main.go

```diff
@@ -228,11 +228,12 @@ func main() {
 	}
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                 scheme.Scheme,
-		Metrics:                metricsServerOptions,
-		HealthProbeBindAddress: probeAddr,
-		LeaderElection:         enableLeaderElection,
-		LeaderElectionID:       "9c4404e7.operatorframework.io",
+		Scheme:                        scheme.Scheme,
+		Metrics:                       metricsServerOptions,
+		HealthProbeBindAddress:        probeAddr,
+		LeaderElection:                enableLeaderElection,
+		LeaderElectionID:              "9c4404e7.operatorframework.io",
```

Comment:
Nit: Could we not use operator-controller-lock here as well like we have in catalogd? 
Maybe for another PR

Reply:
I don't think we can do that. Because then old code and new code could both get elected leader under different leader election IDs.